src/control/server/storage/scm/ipmctl.go

```diff
@@ -563,6 +611,75 @@ func (cr *cmdRunner) createNamespaces() (storage.ScmNamespaces, error) {
 	return cr.getNamespaces()
 }
 
+func (cr *cmdRunner) createDualNamespacesPerNUMA() (storage.ScmNamespaces, error) {
```

Comment:
This doesn't seem like a good long-term approach. As far as I'm aware, there's no reason that we shouldn't support N namespaces per node, where N is evenly-dividable. It's true that the current request is to support two namespaces, but it doesn't seem unreasonable to imagine that future platforms might support 4 or 8.

Reply:
done